index.html

```diff
@@ -281,6 +283,26 @@ <h5><dfn>SmartCardReaderStateFlagsOut</dfn> dictionary</h4>
             </dl>
           </section>
         </section>
+        <section data-dfn-for="SmartCardGetStatusChangeOptions">
+          <h4><dfn>SmartCardGetStatusChangeOptions</dfn> dictionary</h4>
+          <pre class="idl">
+            dictionary SmartCardGetStatusChangeOptions {
+              DOMHighResTimeStamp timeout;
+              AbortSignal signal;
+            };
+          </pre>
+
+          <dl>
+            <dt><dfn>timeout</dfn> member</dt>
+            <dd>Timeout parameter for the GetStatusChange() [[PCSC5]] method. If
```

Comment:
Follow-up: When specifying the algorithm for this method make sure to mention that the `timeout == 0` case is special.